packages/block-editor/src/hooks/background.js

```diff
@@ -63,7 +60,7 @@ export function setBackgroundStyleDefaults( backgroundStyle ) {
 	let backgroundStylesWithDefaults;
 
 	// Set block background defaults.
-	if ( !! backgroundImage?.url ) {
+	if ( !! backgroundImage?.url || typeof backgroundImage === 'string' ) {
```

Comment:
Note to self, check if this is still required. 

It's supposed to include `backgroundImage: url('')` values.

Reply:
I don't think so as `backgroundImage` with string values will inevitably come from theme.json. 

Given the logic in https://github.com/WordPress/gutenberg/pull/64192, default values shouldn't be applied to those values.